### PR TITLE
1.2.0 Version bump and readme.txt update

### PR DESCRIPTION
**Pull request by matiasbenedetto**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ Donate link: https://automattic.com/
 Tags: themes, theme, block-theme
 Requires at least: 6.0
 Tested up to: 6.0
-Stable tag: 1.1.3
+Stable tag: 1.2.3
```
Comment (mikachan) [inline readme.txt]: Should this be 1.2.0?

Comment (matiasbenedetto) [reply readme.txt]: right! I corrected it to 1.2.0

File: readme.txt [hunk 2]
```diff
@@ -14,26 +14,33 @@ A WordPress plugin to create block themes.
...
+- Embed Google font in your current theme
+- Embed local font file assets
 
-Select one of the available options and then, if necessary, add the details for the theme here. These details will be used in the style.css file.
+= Step 2 – Style Customizations =
+Make changes to your site styles and templates using the Site Editor. You can also include new fonts using the plugin options.
 
-Click "Create theme” button, to save the theme containing all the files.
+= Step 3 – Export =
+Still in the WordPress dashboard, navigate to "Appearance" -> "Create Block Theme" section. Select one of the available options and then, if necessary, add the details for the theme here. These details will be used in the style.css file. Click "Create theme” button, to save the theme.
```
Comment (mikachan) [inline readme.txt]:
```suggestion
Still in the WordPress dashboard, navigate to "Appearance" -> "Create Block Theme" section. Select one of the available options and then, if necessary, add the details for the theme here. These details will be used in the style.css file. Click "Generate” button, to save the theme.
```

I've just realised, as part of https://github.com/WordPress/create-block-theme/pull/112, this reference to the button needs to be updated! It probably needs updating in README.md as well.

Comment (matiasbenedetto) [reply readme.txt]: Good catch. Commited your sugestion

Comment (mikachan) [reply readme.txt]: I've changed this in README.md [here](https://github.com/WordPress/create-block-theme/pull/116/commits/bdf4510b5bfa9941fb18425f5a51bca773394ff9) as well.

**Review by mikachan — APPROVED** [review]
LGTM!

**Pull request merged**